release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -57,6 +57,21 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-8-security"]
 === Security and compliance
 
+[id="ocp-4-8-security-wildcard-subjects-headless-services"]
+==== Wildcard subject for service serving certificates for headless services
+
+Generating a service service certificate for headless services now includes a wildcard subject in the format of `*.<service.name>.<service.namespace>.svc`. This allows for TLS-protected connections to individual stateful set pods without having to manually generate certificates for these pods.
```

Comment:
"service service certificate" -- should this be "service serving certificate" or perhaps just "service certificate"?

Reply:
You're right, thanks for catching that! It's a mouthful :)